src/frontend/src/flows/authorize/index.ts

```diff
@@ -7,33 +10,50 @@ import { recoveryWizard } from "../recovery/recoveryWizard";
 import { authenticationProtocol } from "./postMessageInterface";
 import {
   authenticateBox,
-  AuthTemplates,
+  AuthnTemplates,
 } from "../../components/authenticateBox";
 
 /* Template for the authbox when authenticating to a dapp */
-export const authTemplatesAuthorize = ({
+export const authnTemplateAuthorize = ({
   origin,
   derivationOrigin,
 }: {
   origin: string;
   derivationOrigin?: string;
-}): AuthTemplates => {
-  const message = html`<p class="t-lead">
-    Connect to<br />
-    <span class="t-strong">${origin}</span><br />
-  </p>`;
+}): AuthnTemplates => {
   const chasm =
     derivationOrigin !== undefined
-      ? {
+      ? mkChasm({
           info: "shared identity",
           message: html`<span class="t-strong">${origin}</span> is an
             alternative domain of <br /><span class="t-strong"
               >${derivationOrigin}</span
             ><br />and you will be authenticated to both with the same identity.`,
-        }
+        })
       : undefined;
 
-  return { message, chasm, button: "Authorize" };
+  const wrap = (slot: string) => html`
+    <div class="l-stack t-centered">
+      <p class="t-lead">
+        ${slot}<br />
+        <span class="t-strong">${origin}</span><br />
+      </p>
+      ${chasm}
+    </div>
+  `;
+  return {
+    new: {
+      slot: wrap("Create an anchor to continue to"),
```

Comment:
as you know, frenglish is my strength, but doesn't "...to continue to..." sound a bit redundant or repetitive?